ouroboros-consensus-cardano/CHANGELOG.md

```diff
@@ -17,3 +17,29 @@ If you have any doubts, please consult the [release
 process](../ouroboros-consensus/docs/ReleaseProcess.md).
 
 # Changelog entries
+
+<a id='changelog-0.2.0.0'></a>
+## 0.2.0.0 — 2023-02-09
+
+### Patch
+
+- Remove redundant proxy argument for `ledgerDbTip`.
+
+### Non-Breaking
+
+- Adapted to the new reorganization of Mempool modules in `ouroboros-consensus`.
```

Comment:
We use the present tense for the `Patch` and past tense here. Maybe we should use one, and document the decision in the release process doc.